coturn/coturn/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '2'
+services:
+  gateway:
+    build: gateway
+    ports:
+      - "8000:8000"
+    hostname: "kaazing.example.com"
```

Comment:
I thought that it is enough to add `<feature.turn.rest>true</feature.turn.rest>` in the xml that is copied in the container (which I did).